### ISSUE TITLE
base64 vs. base64url encoding in id_token

### ISSUE THREAD

**Issue by ratzepue** [nicko170/wp-openid#4]
First of all, thank you for providing this plugin!

I've had an issue in function login_callback(), line 242. The command

`json_decode(base64_decode(explode('.', $token['id_token'])[1]), true);`

returns NULL in cases where my ID token contains the characters '-' or '_', which are part of the base64url character set. The function `base64_decode()` returns gibberish after the unexpected character, in these cases.

The base64url encoding in the ID token seems to be correct, according to https://openid.net/specs/openid-connect-core-1_0.html#id_tokenExample: "_The value of the id_token parameter is the ID Token, which is a signed JWT, containing three base64url-encoded segments ..._"

In summary, I think the id_token should be treated as being encoded with base64url.
